validation-test/radar/r118770461.swift

```diff
@@ -0,0 +1,8 @@
+@usableFromInline
+struct Foo {
+    var integer: Int
+}
+
+struct Bar: Sendable {
+    var foo: Foo // warning: Stored property 'foo' of 'Sendable'-conforming struct 'Bar' has non-sendable type 'Foo'
```

Comment:
This has to be `// expected-warning` to actually check that the warning is produced; and a TODO comment that it should not be once the issue is fixed?